controllers/operands/kubevirt.go

```diff
@@ -294,7 +295,71 @@ func hcWorkloadUpdateStrategyToKv(hcObject *hcov1beta1.HyperConvergedWorkloadUpd
 	return kvObject
 }
 
-func getKVConfig(hc *hcov1beta1.HyperConverged) (*kubevirtcorev1.KubeVirtConfiguration, error) {
+func getRatesFromConfigMap(configMap *corev1.ConfigMap) (*kubevirtcorev1.ReloadableComponentConfiguration, error) {
+
+	qps, ok := configMap.Data["qps"]
+
+	if !ok {
+		return nil, errors.New("qps parameter not found in configmap " + configMap.Name)
```

Comment:
Is it an error case?

Reply:
Maybe an error is not the best approach. The idea here is if one of these parameters is not present in the configmap, we do not set the rate limits. If you have a suggestion about this, it would be much appreciated. Thanks!